src/frontends/onnx/frontend/src/op/random_uniform_like.cpp

```diff
@@ -28,18 +36,14 @@ OutputVector random_uniform_like(const Node& node) {
         target_type = input.get_element_type();
     }
 
-    const auto target_shape = std::make_shared<default_opset::ShapeOf>(input);
+    auto high_convert = std::make_shared<ov::op::v0::Convert>(high_const, target_type);
+    auto low_convert = std::make_shared<ov::op::v0::Convert>(low_const, target_type);
 
-    const auto high_const = node.get_attribute_as_constant<float>("high", 1.0f);
-    const auto low_const = node.get_attribute_as_constant<float>("low", 0.0f);
-    const auto seed = node.get_attribute_value<float>("seed", 0.f);
-
-    const uint64_t global_seed = 0;
-    const auto seed_uint64 = static_cast<uint64_t>(seed * 1000);
+    const auto target_shape = std::make_shared<default_opset::ShapeOf>(input);
```

Comment:
please use the exact version of `ShapeOf` that should be latest one `v3` and include `#include "openvino/op/shapeof.hpp"`

Reply:
Done!